tools/cli/lib/installcmd.c

```diff
@@ -186,6 +186,9 @@ TDNFCliAlterCommand(
                   &nPackageCount);
     BAIL_ON_CLI_ERROR(dwError);
 
+    dwError = TDNFCliRefresh(pContext);
```

Comment:
Why can't all these call to TDNFCliRefresh can be done in one place just befoe: https://github.com/vmware/tdnf/blob/99885bd58326bc9defd0d516f51e55da0adebaa9/tools/cli/main.c#L158